packages/apollo-fragment-react/package.json

```diff
@@ -41,23 +41,23 @@
       "threshold": "1 Kb"
     }
   ],
+  "dependencies": {
+    "@apollo/client": "^3.5.6",
+    "apollo-fragment-utils": "^0.2.1",
```

Comment:
I'll need to bump the version of `apollo-fragment-utils` once it gets published with the changes. Wondering if there's a better way to do that with lerna... or if this has to be a two step operation